manifests/config.pp

```diff
@@ -53,12 +53,10 @@
     ensure => absent,
   }
 
-  $listen_socket = $foreman::foreman_service_bind ? {
-    Stdlib::IP::Address::V6 => "[${foreman::foreman_service_bind}]:${foreman::foreman_service_port}",
-    default                 => "${foreman::foreman_service_bind}:${foreman::foreman_service_port}",
-  }
-
   if $foreman::use_foreman_service {
```

Comment:
Is there a best practice as to why this section does not live in `service.pp` since they are tied to the service itself?

Reply:
I think initially when I wrote this I considered them config files. Now looking back (and with the experiences from Pulpcore as well) I think `service.pp` would be a better place. For example, there's no reason this needs to be done before the DB, apache etc.